examples/response-with-multiparts/response_with_multiparts.bal

```diff
@@ -119,6 +119,22 @@ function handleContent(mime:Entity bodyPart) {
         } else {
             log:printError("Error in parsing text data", err = payload);
         }
+    } else if (mime:APPLICATION_PDF == baseType) {
+        // [Extracts byte stream](https://ballerina.io/learn/api-docs/ballerina/#/ballerina/http/latest/http/classes/Response#getByteStream) from the body part and save it as a file.
+        var payload = bodyPart.getByteStream();
+        if (payload is stream<byte[], io:Error>) {
+            //Writes the incoming stream to a file using `io:fileWriteBlocksFromStream` API by providing the file location which the content should be written to.
+            io:Error? result = io:fileWriteBlocksFromStream(
+                                    "./files/ReceivedFile.pdf", payload);
+
+            if (result is error) {
+                log:printError("Error occurred while writing ",
```

Comment:
Shall we use consistent cases for these log messages? Above, these log messages started with a lower case 'e'

Reply:
Throughout this BBE, all error messages have started with a capital letter. Hence thought of keeping it as it is.